### PR TITLE
Add release notes for 2.10.2

### PR DESCRIPTION
**Pull request by madkinsz**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Prefect Release Notes
 
+## Release 2.10.2
+
+Fixes a bug where deployments were not downloaded from remote storage blocks during flow runs — https://github.com/PrefectHQ/prefect/pull/9138
+
+### Enhancements
+- Adding httpx.ConnectTimeout to the list of retry exceptions in base client — https://github.com/PrefectHQ/prefect/pull/9125
```
Comment (serinamarie) [inline RELEASE-NOTES.md:8]:
```suggestion
- Add httpx.ConnectTimeout to the list of retry exceptions in base client — https://github.com/PrefectHQ/prefect/pull/9125
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+
+### Contributors
+- @billpalombi
+* @sorendaugaard made their first contribution in https://github.com/PrefectHQ/prefect/pull/9125
```
Comment (serinamarie) [inline RELEASE-NOTES.md:12]: Should Bill be in here? Maybe Alex?

Comment (madkinsz) [reply RELEASE-NOTES.md]: Thanks!

**Review by serinamarie — APPROVED** [review]
LGTM :)

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+
+Fixes a bug where deployments were not downloaded from remote storage blocks during flow runs — https://github.com/PrefectHQ/prefect/pull/9138
+
+### Enhancements
+- Add httpx.ConnectTimeout to the list of retry exceptions in base client — https://github.com/PrefectHQ/prefect/pull/9125
+
+### Contributors
+* @sorendaugaard made their first contribution in https://github.com/PrefectHQ/prefect/pull/9125
+
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:13]:
```suggestion

```

**Pull request merged**